### DISCUSSION TITLE
Unit testing HTML report JS

### DISCUSSION THREAD

**Discussion by TroyWalshProf** [TroyWalshProf/SeleniumAxeDotnet#183] [General]
@jonreding2010  I am not 100% sure we care about adding unit tests to the JS file as outlined in #180.

@dbjorge, mind weighing in here?
My gut is to just exclude the file, but would like a second opinion.

**Comment by dbjorge** [comment]
I don't see any reason we couldn't/wouldn't want to write unit tests for those. That said, I don't think we need to block "moving them into a separate file" on having unit tests; the js already doesn't have unit tests, moving it to a separate file isn't regressing that. I'd be fine with adding an exclusion for the js file for the purposes of the PR moving it out.
